tests/test_signal_processing.py

```diff
@@ -0,0 +1,114 @@
+"""Tests of `signal_processing` for pynapple"""
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import pynapple as nap
```

Comment:
If we are trying to match any other implementation (like pywavelet or similar) we should have a test that checks that the output of the functions we are matching are actually the same.

Reply:
Because of some changes in logic e.g. using different convolve functions, the pywt wavelet decomposition and ours do not function exactly the same any more, and will not produce identical output.